.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -67,8 +67,8 @@ body:
         Copy this command and run it in a terminal. It will open a text editor with the system information and logs. Paste the output below. Redact any sensitive information from the logs.
         ```bash
         TMPFILE=$(mktemp authd-system-info-XXXXXX.md)
-        if command -v xdg-open > /dev/null && [ -n "${DISPLAY:-}" ]; then EDITOR=xdg-open; elif [ -z "${EDITOR:-}" ]; then EDITOR=editor; fi 
-        cat > "$TMPFILE" <<EOF && $EDITOR "$TMPFILE" && rm -f "$TMPFILE"
+        if command -v xdg-open > /dev/null && [ -n "${DISPLAY:-}" ]; then authd_editor=xdg-open; elif [ -n "${EDITOR:-}" ]; then authd_editor=$EDITOR; else authd_editor=editor; fi
```

Comment:
don't need the `else` thingy, just use `${authd_editor:-editor}` when calling it